src/Controller/ProductController.php

```diff
@@ -62,20 +72,23 @@ public function show(Product $product): Response
     }
 
     /**
-     * @Route("/{id}/edit", name="product_edit", methods={"GET","POST"})
+     * @Route("/{id}/edit", name="product_edit", methods={"GET","PATCH"})
      */
     public function edit(Request $request, Product $product): Response
     {
-        $form = $this->createForm(ProductType::class, $product);
+        $form = $this->createForm(ProductType::class, $product, ['method' => 'PATCH']);
```

Comment:
Dlaczego zmieniłaś metodę na PATCH? Nie mówię, że to jest źle, ale to zależy od kontekstu, więc poproszę o wyjaśnienie